### ISSUE TITLE
support internationalization

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-atag-report-tool#42]
I noticed that some labels/dialogs/texts are embedded in the content, which may make this tool harder to translate into other languages? Would be good to have just a single JSON or other config file that people can translate without needing to fiddle around with any source code.

**Comment by hidde** [comment]
Yes, great idea, and I feel something we should/can do early, now that we don't have many strings yet.

I'll double check with other WAI tools on how they approach it.

**Comment by slhenry** [comment]
+ for building in flexibility for translation just in case...

(yet, given all data, translation is highly unlikely in the "foreseeable future" given workload and other priorities)

**Comment by hidde** [comment]
For the 0.3 milestone I have spent some time on this issue, reading up on the different approaches we could have here. Actual implementation will have to wait to a later version (and probably await future decisions on translation of the WCAG EM Report Tool).
